### PR TITLE
Update docs

### LINKED ISSUE #104: Update user and dev guide again
Shifted to #149

### PR DESCRIPTION
**Pull request by INCENDE**
Shifted the to-dos here. Fixes #104 
- [x] User guide should have clear correspondence to user stories
- [x] Insight from product survey
- [x] More NFRs than the handbook
- [x] Make it more informal like 2101, directly address the user
- [x] Do not use terms unfamiliar to user e.g. floating task
- [x] Introduction: each task has a index... before describing deleting tasks
- [x] More examples
- [x] App behavior: How the app responds so that the user can verify his/her action was correct by comparing the actual app behavior to the expected behavior.
- [x] UI screenshots
- [x] Add examples to command summary
- [x] Content page

Developer Guide
- [x] Compare with other alternative designs possible.
- [x]  Add more descriptions e.g. show at least one interaction (i.e. sequence diagrams) within each component. But this doesn't mean to add more verbose text about trivial and unstable things.
- [x]  Use object diagrams to show the object structure at runtime
- [x]  Use activity diagrams and OO domain models where applicable (covered in yesterday's lecture)
- [x]  Explain how they implemented things such as undo/redo (since they weren't given features in AddressBook Level 4)
- [x]  explain other interesting design decisions you made.
- [x] sequence diagram for other command e.g. add?

### CODE REVIEW

**Comment by rachx** [comment]
View latest user guide here: https://github.com/CS2103AUG2016-W11-C2/main/blob/8a991955c5d38423ba22666de3e5049ce27ddc8f/docs/UserGuide.md

Issues

@fanwgwg 
- [ ] Update all screenshots after changing UI color
- [ ] Add screenshots for help window
- [ ] In UI walkthrough, change the colors described

@burnflare 
- [ ] Ensure that our app only allow (by time) or (from time to time) i.e. complains about incorrect command format when all 3 keywords are present, or either from/to is present only
(Described in guide as a task cannot have a deadline and a event time at the same time
- [ ] Check date time format in user guide vs app behavior

@INCENDE 
- [ ] read and check again since I changed the header and order (thanks!)
- [ ] add more whitespace between different command
- [ ] Can you update "how to transfer my data to another computer" or just remove it?

@rachx
- [ ] Add examples to command summary table

**Comment by rachx** [comment]
Can the test script be updated again?

**Comment by rachx** [comment]
find diagram in user guide is wrong. please fix.

**Comment by rachx** [comment]
Fixes #199

**Comment by INCENDE** [comment]
@rachx could you fix the conflicts?

**Comment by burnflare** [comment]
Can we merge this in? Getting hard to review the updated docs when this PR is so large

**Comment by INCENDE** [comment]
Yes please do.

**Comment by burnflare** [comment]
Actually I can read now properly after switching branches in the main interface https://github.com/CS2103AUG2016-W11-C2/main/tree/update-guides-for-v0.4/docs

**Comment by INCENDE** [comment]
Will fix the conflicts now

**Comment by INCENDE** [comment]
Can we wrap this up by 9pm? Will do a final check before transferring to doc file

**Comment by rachx** [comment]
@burnflare can you remove all the collate files from this pr and collate again in a separate branch after this is merged?

**Comment by burnflare** [comment]
Pushed a commit that removes everything to do with `collated/`

**Comment by rachx** [comment]
Updated about us again (add more features and change the links to match the sub-headers of the new user guide))

**Comment by rachx** [comment]
Tick if you are okay with the files and ready to merge.

Rachael
- [x] User guide
- [x] Developer guide - think the justification for singleton/ add, schedule use cases should be updated but not changing it.

Weiguang
- [ ] User guide
- [ ] Developer guide

Justin
- [x] User guide
- [x] Developer guide

Vishnu
- [x] User guide
- [x] Developer guide

File: src/main/java/seedu/agendum/ui/UiPart.java [hunk 1]
```diff
@@ -74,7 +75,8 @@ protected Stage createDialogStage(String title, Stage parentStage, Scene scene)
      * @param iconSource e.g. {@code "/images/help_icon.png"}
      */
     protected void setIcon(String iconSource) {
-        primaryStage.getIcons().add(AppUtil.getImage(iconSource));
+//        primaryStage.getIcons().add(AppUtil.getImage(iconSource));
+        primaryStage.getIcons().add(new Image(this.getClass().getResourceAsStream(iconSource)));
```
Comment (burnflare) [inline src/main/java/seedu/agendum/ui/UiPart.java:78]: Why is there commented code here?

Comment (burnflare) [reply src/main/java/seedu/agendum/ui/UiPart.java:78]: @fanwgwg

**Review by burnflare — APPROVED** [review]
To merge after addressing commented code

**Comment by rachx** [comment]
should we add sync interface in the diagram?

File: src/main/java/seedu/agendum/ui/UiPart.java (hunk 1 above)
Comment (burnflare) [reply src/main/java/seedu/agendum/ui/UiPart.java:78]: Also, what does this code change?

Comment (fanwgwg) [reply src/main/java/seedu/agendum/ui/UiPart.java:78]: O forgot to delete it !!!

Comment (burnflare) [reply src/main/java/seedu/agendum/ui/UiPart.java:78]: Thanks! 👍

**Comment by burnflare** [comment]
@rachx To the Model diagram? Can you help me do it, I don't have PPT installed. 🎱

**Pull request merged**